presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
 
-* ``task.info-refresh-max-wait``:
-  Controls staleness of task information, which is used in scheduling. Increasing
-  this value can reduce coordinator CPU load, but may result in suboptimal
-  split scheduling.
-
-* ``task.max-worker-threads``:
-  Sets the number of threads used by workers to process splits. Increasing this number
-  can improve throughput, if worker CPU utilization is low, but will cause increased
-  heap space usage.
-
-* ``distributed-joins-enabled``:
-  Use hash distributed joins instead of broadcast joins. Distributed joins
-  require redistributing both tables using a hash of the join key. This can
-  be slower (sometimes substantially) than broadcast joins, but allows much
-  larger joins. Broadcast joins require that the tables on the right side of
-  the join fit in memory on each machine, whereas distributed joins only
-  need to fit in distributed memory across all machines. This can also be
-  specified on a per-query basis using the ``distributed_join`` session property.
-
-* ``node-scheduler.network-topology``:
-  Sets the network topology to use when scheduling splits. "legacy" will ignore
-  the topology when scheduling splits. "flat" will try to schedule splits on the same
-  host as the data is located by reserving 50% of the work queue for local splits.
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++              property name               +                   type                   +           default value            +tunning the property                                   +
++==========================================+==========================================+====================================+=======================================================+
++      ``task.info-refresh-max-wait``      +          ``String`` (duration)           +             ``200 ms``             +Controls staleness of task information which is used   +
++                                          +                                          +                                    +in scheduling. Increasing this value can reduce        +
++                                          +                                          +                                    +coordinator CPU load but may result in suboptimal      +
++                                          +                                          +                                    +split scheduling.                                      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.max-worker-threads``        +               ``Integer``                +    ``Number of node CPU's * 4``    +Sets the number of threads used by workers to process  +
```

Comment:
Is this "default value" somehow auto-tuned? I.e., does Presto discover the number of CPU cores? If not, what is the mechanism to set the value initially?